golang/README.md

```diff
@@ -0,0 +1,23 @@
+# Training Session in Golang
+
+## Intro
+
+The training in Golang will cover all the concepts you need to be aware of to start using IOTICS. In particular, it will be divided into:
+- Morning Session
+  - Create a Twin Receiver
+  - Create a Twin Sender
+- Afternoon Session
+  - Create a Publisher Connector
+  - Create a Synthesiser Connector
+
+More details will be given on the day of the training. For now, we only need you to setup your laptop so you come prepared for the code exercises.
+
+## Laptop Setup
+
+1.  Install [Golang](https://go.dev/doc/install) on your machine;
+2.  Clone this repository;
+3.  Install the required dependencies. Navigate to the golang folder and run:
+```bash
+go mod init iotics-training
+go mod tidy
```

Comment:
`go mod tidy` may behave differently depending on the version in use.
I have 1.17 locally and `go mod tidy` requires the `-compat=1.17` flag to behave correctly in this case.

Reply:
What do you suggest ?

Reply:
Sure. Now I'm aware that different Go versions may behave differently